model/state_model_nlparam.jl

```diff
@@ -140,14 +145,14 @@ replace_nan_inf(theta_n)
 replace_nan_inf(theta_m)
 
 #Substitution and transformation elasticities
-@NLparameter(cge, es_va[r in regions, s in sectors] == 1); # value-added nest - substitution elasticity
-@NLparameter(cge, es_y[r in regions, s in sectors] == 0); # Top-level Y nest (VA,M) - substitution elasticity
-@NLparameter(cge, es_m[r in regions, s in sectors] == 0); # Materials nest - substitution elasticity
-@NLparameter(cge, et_x[r in regions, g in goods] == 4); # Disposition, distribute regional supply to local, national, export - transformation elasticity
-@NLparameter(cge, es_a[r in regions, g in goods] == 0); # Top-level A nest for aggregate demand (Margins, goods) - substitution elasticity
-@NLparameter(cge, es_mar[r in regions, g in goods] == 0); # Margin supply - substitution elasticity
-@NLparameter(cge, es_d[r in regions, g in goods] == 2); # Domestic demand aggregation nest (intranational) - substitution elasticity
-@NLparameter(cge, es_f[r in regions, g in goods] == 4); # Domestic and foreign demand aggregation nest (international) - substitution elasticity
+@NLparameter(cge, es_va[r in regions, s in sectors] == SUB_ELAST[:va]); # value-added nest - substitution elasticity
```

Comment:
For the time being, I think this is a good solution for the substitution elasticities.

### Responses:

1. Ideally, these values can be specified using econometric estimates from the literature. Sometimes the literature produces a range of values. Other times, no estimates exist. It can be useful to adjust elasticity parameters to see how sensitive the model is to a given value. In many cases, a range of scenarios can be included in the results of a research paper to clarify the intuition behind changing the parameter. I'll pass along a recent paper where I perform this type of scenario analysis. More complex methods involve sampling random elasticity values within a given range and then performing a regression to determine how sensitive model output is to a change in that parameter (e.g. monte-carlo). EX: Given a percentage change in a given elasticity, what is the percentage change in emissions, output, welfare, jobs, etc.

2. I am not sure, but this is a good idea to double check. I think I was consistent where necessary, but most don't have direct overlap. The naming convention should mostly refer to aggregated combinations of the keys in ```d``` or ```sld```.